EmotiBit.cpp

```diff
@@ -1058,7 +1056,15 @@ uint8_t EmotiBit::update()
 			writeSerialData(_serialData);
 		}
 	}
-
+	// if not in debug mode, always clear the input serial buffer
+	else
+	{
+		while (Serial.available())
+		{
+			Serial.read();
+		}
+		serialPrevAvailable = 0; // set Previously available to 0
+	}
```

Comment:
This should solve the un-responsive behaviour of the serial in `non debugMode`.
in nonDebug mode, if serial is available, we print hi and clear the whole input buffer.